### DISCUSSION TITLE
Request for review of project Parcel.money

### DISCUSSION THREAD

**Discussion by sriramkasyap** [WalletConnect/walletconnect-monorepo#2148] [Developer Support]
Hi,

We are Parcel (https://parcel.money), a Payments application built on top of Gnosis safe. We have been live for 2 years now and being used by prominent DAOs in the industry like ENS, dYdX and DeveloperDAO. We are trying to upgrade to Walletconnect V2 and have submitted our project to review on Waleltconnect Cloud. Request you to review at the earliest so that our users can experience the updated signing experience. 

uuid: 87855ec6-15d1-4bb6-b09e-0b97eebac5ec

**Comment by finessevanes** [comment]
@sriramkasyap I took a look at your project. 2 things:
1. I go to parcel.money, click to connect with WalletConnect but I'm seeing the old modal and the URI is still our v1: `wc:c7fbb1c2-6e7f-45cb-8e3e-5931b736c410@1?bridge=https%3A%2F%2F5.bridge.walletconnect.org&key=906754322f39519bd7d7ee244bbbd6108f41fd1c31f5c441b221da60dd7b409e`
2. Can you please update our name in your wallet selection options from `Wallet Connect` to `WalletConnect`?

**Reply by sriramkasyap** [reply]
Hi @finessevanes , thank you for taking a look at our project. 

The app you reviewed is the current, live project and is using walletconnect v1.
However, We are currently in the process of releasing some new features of the application for our private partners where we would like to implement V2 of walletconnect. I couldn't share the preview link earlier due to some logistic issues. 

I have now updated the form with the latest preview link and details for your review. Request you to take a look at the new app links.

**Reply by sriramkasyap** [reply]
Hi @finessevanes , Can you please provide us an update here?

**Reply by sriramkasyap** [reply]
Hi @finessevanes , Could you please let us know if there is any progress on this?

**Comment by sriramkasyap** [comment]
Hey @finessevanes , Do you need any more information from our side to help process this? Please let us know as this is blocking us from integrating with Walletconnect V2.

**Reply by finessevanes** [reply]
I edited the submission to only include sign v1 support since you don't have v2 support. Once v2 is in prod, you can resubmit the application and toggle v1 off. It has been approved.
